tests/integrations/sanic/test_sanic.py

```diff
@@ -8,12 +8,20 @@
 
 from sentry_sdk import capture_message, configure_scope
 from sentry_sdk.integrations.sanic import SanicIntegration
+from sentry_sdk.tracing import TRANSACTION_SOURCE_COMPONENT, TRANSACTION_SOURCE_URL
 
 from sanic import Sanic, request, response, __version__ as SANIC_VERSION_RAW
 from sanic.response import HTTPResponse
 from sanic.exceptions import SanicException
 
+from sentry_sdk._types import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from collections.abc import Iterable, Container
+    from typing import Any, Optional
```

Comment:
The tests are not checked with mypy so these (and the associated function annotations) are not necessary, though you can keep them if you want :)

Reply:
I added them so I could get type hints in VSCode while editing. So if it's okay, I would like to keep them